doc QBFT and update consensus docs (#115)
ConsenSys/doc.goquorum: docs/Concepts/Consensus/Comparing-PoA.md

---
description: Proof of Authority consensus protocols comparison
---

# Comparing Proof of Authority consensus protocols

GoQuorum implements the [IBFT](../../HowTo/Configure/Consensus-Protocols/IBFT.md), [QBFT](../../HowTo/Configure/Consensus-Protocols/QBFT.md),
[Raft](../../HowTo/Configure/Consensus-Protocols/Raft.md), and [Clique](../../HowTo/Configure/Consensus-Protocols/Clique.md)
Proof of Authority (PoA) consensus protocols.
PoA consensus protocols work when participants know each other and there is a level of trust between them (for example,
in a permissioned consortium network).

PoA consensus protocols have faster block times and a much greater transaction throughput than the Ethash Proof of Work
consensus protocol used on Ethereum Mainnet.

In the GoQuorum PoA consensus protocols, a group of nodes in the network act as validators (IBFT and QBFT), verifiers (Raft)
or signers (Clique).

> The existing nodes in the validator/verifier/signer pool vote to add nodes to or remove nodes from the pool.

Reviewer: ```suggestion
Existing validators, verifiers, or signers vote to add or remove network nodes.
```
